apps/l3dtool/l3dtool.cpp

```diff
@@ -781,6 +814,16 @@ int ExtractFile(const Arguments::Extract& args)
 	memcpy(indexBuffer.data.data(), l3d.GetIndices().data(), sizeOfIndices);
 	gltf.buffers.push_back(indexBuffer);
 
+	// TODO(raffclar): tiny_gltf has not yet added support for serialising binary data. Using strings with base64
+	std::map<std::string, tinygltf::Value> extras;
+	extras["footprint"] =
+	    tinygltf::Value(tinygltf::base64_encode(l3d.GetFootprintData().data(), l3d.GetFootprintData().size()));
+	extras["uv2"] = tinygltf::Value(tinygltf::base64_encode(l3d.GetUv2Data().data(), l3d.GetUv2Data().size()));
```

Comment:
Uv clearly should not be in extras. They should probably be in a buffer with an accessor, view.
You'll need to check the format of the data but `type = "VEC2, componentType = GL_FLOAT` is very likely.
Attribute can be `TEXCOORD_1`

Reply:
The UV2 buffer is optional and is not present in most meshes. I don't know what the UV2 data currently looks like

Reply:
Are you going to keep it in extras then? If so, add the same issue number as above

Reply:
static_cast<uint32_t>(l3d.GetUv2Data().size())